index.d.ts

```diff
@@ -52,6 +52,7 @@ declare module 'warframe-items' {
         description?: string;
         releaseDate?: string;
         excludeFromCodex?: boolean;
+        masterable?: boolean;
```

Comment:
given that it's applying on all items, shouldn't be optional, like it should fail types if one of the items is missing this altogether...

```suggestion
        masterable: boolean;
```